hoot-core/src/main/cpp/hoot/core/algorithms/rubber-sheet/RubberSheet.h

```diff
@@ -75,7 +75,7 @@ class RubberSheet : public OsmMapOperation
 
   RubberSheet();
 
-  void apply(boost::shared_ptr<OsmMap>& map);
+  void apply(boost::shared_ptr<OsmMap>& map) override;
```

Comment:
All of the `OsmMapOperation::apply` overridden functions need to be `virtual` like this one here.